docs/governance/working-groups/recommendations/policies-recommendations.md

```diff
@@ -0,0 +1,24 @@
+# Policies & Recommendations
+
+There are two types of guidliness that will be issued to govern the CCC [WG]s.
+
+## Policies
+
+Policies are guidelines which have been proposed by the community structure [WG] and which have been approved by [SC].
```

Comment:
We can link to the Community Structure WG Charter once it has been approved and merged.